File: UserInfo001/app/src/main/java/ru/itmo/userinfo001/MainActivity.java
package ru.itmo.userinfo001;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;
import android.view.View;
import android.widget.Button;
import android.widget.EditText;
import android.widget.TextView;
import android.widget.Toast;

import java.util.Calendar;

public class MainActivity extends AppCompatActivity {

    TextView tvName, tvYear;
    EditText etName, etYear;
    Button button;
    String stName, stYear;
    int year, myYear, myAge;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        tvName = findViewById(R.id.tvName);
        tvYear = findViewById(R.id.tvYear);
        etName = findViewById(R.id.etName);
        etYear = findViewById(R.id.etYear);
        button = findViewById(R.id.button);
    }

    public void goDoIt(View view) {

        stName = etName.getText().toString();
        stYear = etYear.getText().toString();

        if (stName.equals("")) return;

        year = Calendar.getInstance().get(Calendar.YEAR);
        myYear = Integer.parseInt(stYear);
        if (myYear > year) return;

        myAge = year - myYear;
        Toast.makeText(this, stName + ", your age " + myAge + " years", Toast.LENGTH_LONG).show();

    }
}

File: Hometask1ForChapter5/app/src/main/java/ru/itmo/hometask1forchapter5/MainActivity.kt
package ru.itmo.hometask1forchapter5

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.view.View
import android.widget.Button
import android.widget.EditText
import android.widget.TextView

class MainActivity : AppCompatActivity() {

    lateinit var etNumOne: EditText
    lateinit var etNumTwo: EditText
    lateinit var textView: TextView
    lateinit var button: Button
    lateinit var stNumOne: String
    lateinit var stNumTwo: String


    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        etNumOne = findViewById(R.id.etNumOne)
        etNumTwo = findViewById(R.id.etNumTwo)
        textView = findViewById(R.id.textView)
        button = findViewById(R.id.button)

    }

    fun ClickButton(view: View) {

        stNumOne = etNumOne.text.toString()
        stNumTwo = etNumTwo.text.toString()

        var myNumOne = Integer.parseInt(stNumOne)
        var myNumTwo = Integer.parseInt(stNumTwo)

        when {
            myNumOne == myNumTwo -> textView.text = "Number1 = Number2"
            myNumOne > myNumTwo -> textView.text = "Number1 > Number2"
            else -> textView.text = "Number2 > Number1"
        }

    }
}